src/stores/orderListStore.ts

```diff
@@ -37,7 +37,8 @@ class OrderListStore extends BaseStore {
     return this.selectedOrder;
   }
 
-  @observable private orders: OrderModel[] = [];
+  @observable hasPendingOrders: boolean = false;
```

Comment:
i would suggest to rename to `hasPendingRequests`. _Pending_ Orders may be confusing and treated as limit orders